integrationconnectors/managedzone/main.tf

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+data "google_project" "target_project" {
```

Comment:
If you want to have two `data.google_project`, one needs to have a `project_id`, otherwise you use the provider's google project (the active project). https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/project

I would suggest adding a comment of the config to help show what you mean (and have a comment if the "test_project" is the "current" project

Reply:
Have added comments